src/form-builder/FullNameField.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { FieldProps } from './types';
+import SelectField from './SelectField';
+import TextField from './TextField';
+
+const FullNameField = (props: FieldProps<string>): JSX.Element => {
+  const suffixes = ['', 'Jr.', 'Sr.', 'II', 'III', 'IV']
+
+  return (
+    <>
+      <TextField
+        id="firstName"
+        name="fullName.firstName"
```

Comment:
Doesn't have to be in this PR, maybe in a future iteration, but it might be cool to allow the user to provide their own field name i.e. veteranFullName, spouseFullName, and then do something like `name='${fieldName}.firstName'`

Reply:
That'll be valuable for sure, I'll push change in this PR itself